libraries/helpers.rb

```diff
@@ -23,12 +23,14 @@ class Chef
   module Rbenv
     module Helpers
       def self.root_path(node, resource_user = nil)
-        node.run_state['root_path'] ||= {}
+        node.run_state['sous-chefs'] ||= {}
+        node.run_state['sous-chefs']['ruby_rbenv'] ||= {}
+        node.run_state['sous-chefs']['ruby_rbenv']['root_path'] ||= {}
 
         if resource_user
-          node.run_state['root_path'][resource_user]
+          node.run_state['sous-chefs']['ruby_rbenv']['root_path'][resource_user]
```

Comment:
```suggestion
          node.run_state['ruby_rbenv']['root_path'][resource_user]
```

Just using ruby_rbenv is probably enough here considering we have the `ruby_rbenv` namespace on supermarket